pallets/pool-registry/src/mock.rs

```diff
@@ -237,6 +240,33 @@ impl<
 	}
 }
 
+pub struct PoolWriteOffPolicyMutateMock<T> {
+	phantom: PhantomData<T>,
+}
+
+parameter_types! {
+	pub const MaxWriteOffPolicySize: u32 = 10;
+}
+
+impl<
+		T: Config
+			+ pallet_pool_registry::Config
+			+ pallet_pool_system::Config<PoolId = u64, Balance = u128, CurrencyId = CurrencyId>,
+	> PoolWriteOffPolicyMutate<<T as pallet_pool_system::Config>::PoolId>
+	for PoolWriteOffPolicyMutateMock<T>
+{
+	type MaxWriteOffPolicySize = MaxWriteOffPolicySize;
+	type Rate = <T as pallet_pool_registry::Config>::Rate;
+	type WriteOffRule = WriteOffRule<Self::Rate>;
+
+	fn update(
+		pool_id: PoolId,
+		policy: BoundedVec<Self::WriteOffRule, Self::MaxWriteOffPolicySize>,
+	) -> DispatchResult {
+		Ok(())
+	}
+}
```

Comment:
I think this trait is the perfect use case for using `mock-builder` and avoiding the need to import a `pallet-loans-ref` dependency in this pallet.